spec/models/generic_file_spec.rb

```diff
@@ -76,8 +76,10 @@
     end
 
     it "should have attached content" do
-      subject.add_file(File.open(fixture_path + '/world.png'), path: 'content', original_name: 'world.png')
-      expect(subject.content).to be_kind_of ActiveFedora::File
+      subject.apply_depositor_metadata('jcoyne')
+      subject.save
```

Comment:
I agree with @cam156 that you have to save before you can attach a directly_contained file, but I don't understand why this test belongs in curation_concerns in the first place.